terraform/modules/iam_role_policy/compliance_role/policy.json

```diff
@@ -6,6 +6,13 @@
           "Effect": "Allow",
           "Resource": "*",
           "Action": [
+            "logs:Cancel*",
+            "logs:Describe*",
+            "logs:Filter*",
+            "logs:Get*",
+            "logs:Start*",
+            "logs:Stop*",
```

Comment:
What's the `Stop` functionality?

Reply:
If you're running a query, and it's taking too long, you want to `Stop` it.

Reply:
`Required to stop a CloudWatch Logs Insights query that is in progress.`

Reply:
from: https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/permissions-reference-cwl.html